http-filter-example/http_filter.cc

```diff
@@ -23,55 +23,57 @@ HttpSampleDecoderFilterConfig::HttpSampleDecoderFilterConfig(
 
 HttpSampleDecoderFilter::HttpSampleDecoderFilter(HttpSampleDecoderFilterConfigSharedPtr config)
     : config_(config) {
-  // populate list of valid operations, implementation will be changed in the next few commits
-  operations_.insert("set-header");
   
   const std::string header_config = headerValue();
 
-  // find the position of the first space character
-  const size_t spacePos = header_config.find(' ');
-  if (spacePos == std::string::npos) {
-    fail("no arguments provided");
-    setError(1);
-    return;
-  }
+  // split by operation (comma delimited config)
+  auto operations = StringUtil::splitToken(header_config, "\n", false, true);
 
-  // can't be a string view
-  const auto operation = header_config.substr(0, spacePos);
+  // process each operation
+  for (auto const& operation : operations) {
+    auto tokens = StringUtil::splitToken(operation, " ");
+    if (tokens.size() < 3) {
+      fail("too few arguments provided");
+      setError(1); // TODO: set error based on globally defined macros
+      return;
+    }
 
-  // error checking -- invalid operation
-  if (operations_.find(operation) == operations_.end()) {
-    fail("invalid operation provided");
-    setError(1);
-    return;
-  }
+    // determine if it's request/response
+    if (tokens[0] != "http-request" && tokens[0] != "http-response") {
```

Comment:
Nitpick: consts for magic strings.
These seem like they would be useful to put in the header, to make accessible to other things

Reply:
Addressed comments in [this PR](https://github.com/DataDog/envoy-filter-example/pull/5)